### PR TITLE
Added selection for schedules

### PR DESCRIPTION
**Pull request by shi24**

### CODE REVIEW

**Review by ryancanavan — APPROVED** [review]

File: DragonLoop/DragonLoopApp/DragonLoopApp/ViewModels/SchedulesViewModel.cs [hunk 1]
```diff
@@ -35,42 +45,58 @@ private async Task ExecuteLoadSchedulesCommand()
                 SchedulesGrid.Children.Clear();
                 SchedulesGrid.RowDefinitions.Clear();
                 SchedulesGrid.ColumnDefinitions.Clear();
+                SchedulesGrid.RowDefinitions.Add(new RowDefinition());
```
Comment (vangoespow) [inline DragonLoop/DragonLoopApp/DragonLoopApp/ViewModels/SchedulesViewModel.cs:48]: Is this in the right place? Is clearing it and then adding a new RowDefinition the way to go?

Comment (ryancanavan) [reply DragonLoop/DragonLoopApp/DragonLoopApp/ViewModels/SchedulesViewModel.cs:48]: +1

Comment (shi24) [reply DragonLoop/DragonLoopApp/DragonLoopApp/ViewModels/SchedulesViewModel.cs:48]: Yes, the Grid Clear() was the commit that @vangoespow added as a hotfix. The new RowDefinition() was initially added after loading Route and Schedule, but I just grouped the Grid Definitions together.

Comment (shi24) [reply DragonLoop/DragonLoopApp/DragonLoopApp/ViewModels/SchedulesViewModel.cs:48]: Capture

Comment (shi24) [reply DragonLoop/DragonLoopApp/DragonLoopApp/ViewModels/SchedulesViewModel.cs:48]: ^ currently the way it is

**Pull request merged**